src/core/shapes/HollowCone.cpp

```diff
@@ -101,40 +100,39 @@ void HollowCone::calculate_dist(const Utils::Vector3d &pos, double &dist,
     time4 = 0.0;
   }
 
-  mdst0 =
+  auto const mdst0 =
       x_2D * x_2D + y_2D * y_2D - 2.0 * x_2D * r0 + r0 * r0 +
       r1 * r1 * time1 * time1 + 0.25 * w * w +
       (-2.0 * y_2D * r1 * time1 + (x_2D - r0) * w) * cos(alpha) -
       (2.0 * x_2D * r1 * time1 - 2.0 * r0 * r1 * time1 + y_2D * w) * sin(alpha);
-  mdst0 = sqrt(mdst0);
-
-  mdst1 = x_2D * x_2D + y_2D * y_2D - 2.0 * x_2D * r0 + r0 * r0 +
-          r1 * r1 * time2 * time2 + 0.25 * w * w +
-          (-2.0 * y_2D * r1 * time2 + (-x_2D + r0) * w) * cos(alpha) +
-          (-2.0 * x_2D * r1 * time2 + 2.0 * r0 * r1 * time2 + y_2D * w) *
-              sin(alpha);
-  mdst1 = sqrt(mdst1);
-
-  mdst2 = x_2D * x_2D + y_2D * y_2D - 2.0 * x_2D * r0 + r0 * r0 + 0.25 * w * w -
-          time3 * w * w + time3 * time3 * w * w +
-          (x_2D - r0) * (-1.0 + 2.0 * time3) * w * cos(alpha) -
-          y_2D * (-1.0 + 2.0 * time3) * w * sin(alpha);
-  mdst2 = sqrt(mdst2);
-
-  mdst3 = x_2D * x_2D + y_2D * y_2D - 2.0 * x_2D * r0 + r0 * r0 + 0.25 * w * w -
-          time4 * w * w + time4 * time4 * w * w -
-          (x_2D - r0) * (-1.0 + 2.0 * time4) * w * cos(alpha) +
-          y_2D * (-1.0 + 2.0 * time4) * w * sin(alpha) + r1 * r1 -
-          2.0 * y_2D * r1 * cos(alpha) +
-          (-2.0 * x_2D * r1 + 2.0 * r0 * r1) * sin(alpha);
-  mdst3 = sqrt(mdst3);
-
-  double distlist[4] = {mdst0, mdst1, mdst2, mdst3};
+
+  auto const mdst1 =
```

Comment:
please don't invest any time in this code... there is an open PR which might make this code go

Reply:
Once you've merged your PR, I'll merge it here to delete the `HollowCone`.

Reply:
done, `HollowCone` is gone